gnovm/tests/files/var21.gno

```diff
@@ -0,0 +1,17 @@
+package main
+
+func foo() (int, bool) {
+	return 1, true
+}
+
+func main() {
+	var a, b = 2, foo()
```

Comment:
can you add this case
```
var a, b, c = 2, foo()
```

Reply:
done here: https://github.com/gnolang/gno/pull/2206/commits/d8236d272f7e7a0cd136b807a2adc7de8e9ac8a0